src/swagger-specs/profile.yaml

```diff
@@ -1,2725 +1,3377 @@
-          type: string
-          required: false
-          x-example: "UPS1"
-        - name: relatedEntityId
-          in: query
-          description: 'ID of the entity that the ExperienceEvents are associated with. Used when looking up ExperienceEvents. For Native XID lookup, use `relatedEntityId=<XID>` and leave `relatedEntityIdNS` absent; For ID:NS lookup, use both `relatedEntityId` and `relatedEntityIdNS` fields.'
-          type: string
-          required: false
-          x-example: "GtghAUFkdGVzdDE"
-        - name: relatedEntityIdNS
-          in: query
-          description: 'Identity Namespace code of the related entity ID of ExperienceEvent. Used when looking up ExperienceEvents. If this field is used, `entityId` cannot be empty.'
-          type: string
-          required: false
-          x-example: "UPS1"
-        - name: fields
-          in: query
-          description: Fields for the model object. By default, all fields will be fetched. Separated by comma.
-          required: false
-          type: string
-          x-example: "person.name.firstName,person.name.lastName"
-        - name: mergePolicyId
-          in: query
-          description: 'ID of the merge policy. A merge policy includes information for Identity stitching and key-value XDM object merging. If not present, the default merge policy will be used.'
-          required: false
-          type: string
-          x-example: 'example-mergePolicy'
-        - name: startTime
-          in: query
-          description: 'Start time of Time range filter for ExperienceEvents. Should be at millisecond granularity. Included. Default: From beginning.'
-          x-example: '1539838505'
-          required: false
+      - $ref: '#/components/parameters/authorization'
+      - $ref: '#/components/parameters/x-api-key'
+      - $ref: '#/components/parameters/x-gw-ims-org-id'
+      - $ref: '#/components/parameters/x-sandbox-name'
+      - name: schema.name
+        in: query
+        description: XED schema class name.
+        required: true
+        schema:
+          type: string
+        example: _xdm.context.profile
+      - name: relatedSchema.name
+        in: query
+        description: If `schema.name` is "_xdm.context.experienceevent", this value
+          must specify the schema for the profile entity that the time series events
+          are related to. For example, `relatedSchema.name=_xdm.context.profile`
+        schema:
+          type: string
+        example: _xdm.context.profile
+      - name: entityId
+        in: query
+        description: ID of the entity. For Native XID lookup, use `entityId=<XID>`
+          and leave `entityIdNS` absent; For ID:NS lookup, use both `entityId` and
+          `entityIdNS` fields.
+        schema:
+          type: string
+        example: GtghAUFkdGVzdDE
+      - name: entityIdNS
+        in: query
+        description: Identity Namespace code. Used for ID:NS lookup. If this field
```

Comment:
Could use a "The" to start the sentences. Should apply to the descriptions in this section